flexget/plugins/filter/crossmatch.py

```diff
@@ -99,10 +102,17 @@ def entry_intersects(self, e1, e2, fields=None):
             log.trace('v1: %r' % v1)
             log.trace('v2: %r' % v2)
 
-            if v1 == v2:
-                common_fields.append(field)
+            if not exact:
+                if v2 in v1:
+                    common_fields.append(field)
+                else:
+                    log.trace('not matching')
```

Comment:
I didn't add the traces, they were in the original file.  Although it is there twice now, since it's in both "in" and ==